### DISCUSSION TITLE
Full AMD support

### DISCUSSION THREAD

**Discussion by yorai1212** [lllyasviel/Fooocus#873] [General]
Is Fooocus going to have full, or a bit more, support?
I hear for Nvidia cards, the generations are much faster, something with Nvidia having CUDA, while AMD having ROCm or DirectLM or something, which is MUCH slower.

Or even slightly more more support for us AMD users. Such as [this](https://github.com/lllyasviel/Fooocus/issues/624#issuecomment-1757327660), so we won't need to change that file every update and such, ya know, just having maybe 2 downloads. One for Nvidia and one for AMD.

The speed of generations is much important to me, personally.
The Collab link is even faster at this point hehe.

Specs:
GPU
AMD Radeon RX 6800 XT

CPU
AMD Ryzen 7 5800X 8-Core Processor

Edit: Another thing, memory issues (or something like that), some generations have a lot of details and such, and I think it takes a lot of VRAM or RAM or whatever it's called, and.. it gives errors sometimes.

**Comment by hqnicolas** [comment]
@yorai1212 Today I create an Repo for AMD RDNA users:
https://github.com/hqnicolas/lllyasvielFooocusROCm
It was tested on Fooocus v2.5.5 for ROCm 
if you like my REPO please leave a star ⭐

**Reply by yorai1212** [reply]
Hey hqnicolas, thank you.
I'm very github-noob. So, I'm not sure how to install the program.

**Reply by hqnicolas** [reply]
Just copy and paste the readme prompts into an clean ubuntu install

**Reply by mashb1t** [reply]
@hqnicolas feel free to provide a PR so your installation instructions can be reviewed and merged into the official Fooocus readme.

**Reply by hqnicolas** [reply]
@mashb1t https://github.com/lllyasviel/Fooocus/pull/2891

**Reply by yorai1212** [reply]
> Just copy and paste the readme prompts into an clean ubuntu install

I think your repo (or fork, not sure how it's called) is just for Linux. While I'm on Windows 10...

**Reply by hqnicolas** [reply]
@yorai1212 I'm not a windows user but you can [try this method](https://www.youtube.com/watch?v=3wofbmwLNUw)

**Reply by yorai1212** [reply]
I don't want to mess with that, sorry. 
Thank you though!
